src/node_snapshotable.cc

```diff
@@ -139,16 +139,15 @@ std::ostream& operator<<(std::ostream& output, const EnvSerializeInfo& i) {
   return output;
 }
 
-class SnapshotSerializerDeserializer {
+class BlobSerializeDeserializer {
  public:
-  SnapshotSerializerDeserializer()
-      : is_debug(per_process::enabled_debug_list.enabled(
-            DebugCategory::MKSNAPSHOT)) {}
+  BlobSerializeDeserializer(bool is_debug_v) : is_debug(is_debug_v) {}
```

Comment:
```suggestion
  explicit BlobSerializeDeserializer(bool is_debug_v) : is_debug(is_debug_v) {}
```
To make the linter happy.